src/components/Dropdown/Dropdown.scss

```diff
@@ -111,7 +111,7 @@
     padding: 0 16px;
   }
 
-  &:before {
+  &::before {
     content: none;
```

Comment:
Is `content: none` the same as `content: ''`? If so, we should use `none` as it's clearer what the intent is.